### PR TITLE
ref(ui): Make stack trace tooltip generic for more issue types

### PR DESCRIPTION
**Pull request by malwilley**
Moves `StackTraceTooltip` into  a parent `GroupPreviewTooltip` component to allow us to add previews for other issue types.

### CODE REVIEW

**Review by evanpurkhiser — APPROVED** [review]

File: static/app/components/groupPreviewTooltip/index.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          issueId={groupId}
+          groupingCurrentLevel={groupingCurrentLevel}
+          eventId={eventId}
+          projectSlug={eventId ? ProjectsStore.getById(projectId)?.slug : undefined}
+        >
+          {children}
+        </StackTracePreview>
+      );
+    default:
+      return <Fragment>children</Fragment>;
```
Comment (scttcper) [inline static/app/components/groupPreviewTooltip/index.tsx:40]: something is off here
```suggestion
      return children;
```

Comment (malwilley) [reply static/app/components/groupPreviewTooltip/index.tsx:40]: Did this for ease of typing, I want to accept a ReactChild, but need to return a JSX.Element or TS will get mad

Comment (scttcper) [inline static/app/components/groupPreviewTooltip/index.tsx:40]: its missing the brackets then right?

```suggestion
      return <Fragment>{children}</Fragment>;
```

Comment (malwilley) [reply static/app/components/groupPreviewTooltip/index.tsx:40]: Ohhh I see, yeah you're right

**Review by scttcper — APPROVED** [review]

**Pull request merged**